pkg/build_zip.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2015 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This tool builds zip files from a list of inputs."""
+
+import argparse
+from datetime import datetime
+from helpers import SplitNameValuePairAtSeparator
+from zipfile import ZipFile, ZipInfo, ZIP_DEFLATED
+
+ZIP_EPOCH = 315532800
+
+def _get_argument_parser():
+  parser = argparse.ArgumentParser(
+    description = 'create a zip file',
```

Comment:
When you can fit on one line, please do so. See the style for the other tools. 
That applies to throughout.

Reply:
Sure, sorry. In the latest push I did my best to match `build_tar.py`.